components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/AuthorizationGrantCache.java

```diff
@@ -107,6 +107,9 @@ public AuthorizationGrantCacheEntry getValueFromCacheByTokenId(AuthorizationGran
     public AuthorizationGrantCacheEntry getValueFromCacheByToken(AuthorizationGrantCacheKey key) {
         AuthorizationGrantCacheEntry cacheEntry = super.getValueFromCache(key);
         if (cacheEntry == null) {
+            if (log.isDebugEnabled()) {
+                log.debug("Getting cache entry from session store using access token");
```

Comment:
Put some context identifier to identify the log.  Ex. key.getUserAttributesId(

Reply:
That AuthorizationGrantCacheKey has either authorization code/ token. That is why I didn't log them